package.json

```diff
@@ -13,7 +13,8 @@
     "format:js:fix": "yarn format:js -w",
     "format:caddy:fix": "caddy fmt --overwrite",
     "build": "docker build . -t lido-content",
-    "start": "docker run -p 3000:3000 lido-content"
+    "start": "docker run -p 3000:3000 lido-content",
+    "start:dev": "docker compose up --watch"
```

Comment:
Feels a bit confusing to have `dev` and `start:dev` 🤔  Shouldn't we just keep one or use different naming?

Reply:
I suppose that `dev` and `start` are used in different situations. We can change this to `dev:docker`. What do you think?

Reply:
done :)